docs/development/test-vectors.rst

```diff
@@ -409,6 +409,8 @@ Custom X.509 Vectors
 * ``ca/ca.pem`` - A self-signed certificate with ``basicConstraints`` set to
   true. Its private key is ``ca/ca_key.pem``. This certificate is encoded in
   several of the PKCS12 custom vectors.
+* ``ca/issuer.pem`` - A certificate signed by ``ca/ca.pem``.
```

Comment:
- This cert is not a CA, is it supposed to be under `ca/`?
- This cert is *issued* by `ca/ca.pem`, it is not an issuer itself, therefore would it not be better to call it `issued.pem` or `issued_by.pem`?

Reply:
Agree. Fixed.